module-3/assignments/activity-2.xsd

```diff
@@ -0,0 +1,39 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
xsd is still an xml, so you need xml declaration here as well
`<?xml version="1.0"?>`